lib/src/value/calculation.dart

```diff
@@ -537,57 +533,44 @@ class SassCalculation extends Value {
   /// arguments is an unquoted `var()` string.
   static Value round(Object strategyOrNumber,
       [Object? numberOrStep, Object? step]) {
+    strategyOrNumber = _simplify(strategyOrNumber);
     numberOrStep = numberOrStep.andThen(_simplify);
     step = step.andThen(_simplify);
-    strategyOrNumber = _simplify(strategyOrNumber);
 
-    var args = [
+    var args = (strategyOrNumber, numberOrStep, step);
+
+    switch (args) {
+      case (SassNumber number, null, null):
+        return SassNumber(number.value.round().toDouble())
+            .coerceToMatch(number);
+
+      case (SassString strategy, SassNumber(), null)
+          when ({'nearest', 'up', 'down', 'to-zero'}.contains(strategy.text)):
+        throw SassScriptException("If strategy is not null, step is required.");
+
+      case (SassNumber number, SassNumber step, null)
+          when (!number.hasUnit('%') && !step.hasUnit('%')):
+        _verifyCompatibleNumbers([number, step]);
+        return number_lib.roundWithStep(SassString('nearest'), number, step);
+
+      case (SassString strategy, SassNumber number, SassNumber step)
+          when (!strategy.isVar && !number.hasUnit('%') && !step.hasUnit('%')):
+        _verifyCompatibleNumbers([number, step]);
+        return number_lib.roundWithStep(strategy, number, step);
+
+      case (Object strategy, Object(), Object())
```

Comment:
Since underscore includes a `null` case and `(Object(), null, null)` would throw an error for any test case where `args.length < 3` and any arg is an unquoted string. Ex.  `a {b: round(test)}` would throw: `Error: test must be either nearest, up, down or to-zero`.

So I had to specify `(Object strategy, Object(), Object())`, instead of `(Object strategy, _, _)`.